### ISSUE TITLE
Unable to Add Kraken API to Hummingbot (Error: Invalid Response from Kraken API)

### ISSUE THREAD

**Issue by Omni-NexusAI** [hummingbot/hummingbot#7337]
### Describe the bug

I am encountering an issue when attempting to connect my Kraken API to Hummingbot. Despite following all the proper steps and verifying my API key settings, I consistently receive an error indicating an invalid response from Kraken. This issue persists even after trying multiple solutions.

**Error Message:**
```
Error: {'error': {'error': [], 'result': {}}}
```

### Steps to reproduce

**Steps to Reproduce:**
1. Start Hummingbot in Docker using:
   ```bash
   docker run -it --name hummingbot-instance -v $(pwd)/hummingbot_files:/conf hummingbot/hummingbot:latest
   ```
2. Use the `connect` command to add Kraken as an exchange:
   ```bash
   connect kraken
   ```
3. Enter the Kraken API key and secret when prompted.
4. Select the appropriate API Tier (e.g., Starter, Intermediate, Pro).
5. Observe the error message.

**What I Have Tried:**
1. Verified API key and secret were copied correctly, with no extra spaces.
2. Enabled the necessary permissions in the Kraken API settings:
   - Query Funds
   - Query Open Orders
   - Place Orders
3. Increased the Nonce Window to 200 in the Kraken API settings.
4. Remade new API keys and secret keys multiple times.
5. Verified my Kraken account tier (Intermediate).

**Expected Behavior:**
Hummingbot should successfully connect to Kraken and retrieve account details like balances.

**Actual Behavior:**
The connection attempt fails, and the error message does not provide sufficient details to identify the cause.

**Additional Context:**
- Other exchanges (e.g., Coinbase) connect without issues.
- Logs indicate an issue with the API response, but the details are unclear.

**Steps to Fix/Feature Requests:**
1. Investigate why Hummingbot is failing to authenticate Kraken API keys.
2. Provide more detailed error messages when the API connection fails (e.g., HTTP response codes or Kraken error codes).
3. Ensure compatibility with the latest Kraken API changes, if applicable.

### Release version

Hummingbot Version: 2.1.0

### Type of installation

Docker

### Attach required files

[errors.log](https://github.com/user-attachments/files/18138667/errors.log)
[logs_hummingbot.log](https://github.com/user-attachments/files/18138668/logs_hummingbot.log)

**Comment by david-hummingbot** [comment]
Just did a quick test following your provided steps to reproduce and I'm able to connect to Kraken without any issues

![image](https://github.com/user-attachments/assets/6dbdb5a5-a9f6-417d-a5ce-47c1f9fa074d)

I checked the log files you sent and there's a bunch of Pyperclip errors - how are you pasting your API keys? 

Also, there's another error that might be password related. Try creating a new folder then run the docker run command again and set a new password (like "a" for example) and see if that helps.

**Comment by Omni-NexusAI** [comment]
I also tried adding the Kraken key on the hummingbot dashboard in a different docker instance and I still got this same error. I'm not sure what it is. I doubt Kraken is generating faulty keys on my account for whatever reason. I just copy and paste using shift+right mouse click and then enter, and for some reason ctr+v or just right click doesn't work let it normally would in the CLI, but when running hummingbot in the ubuntu CLI I get those pyperclip errors.
![Screenshot 2024-12-17 114147](https://github.com/user-attachments/assets/92362ed4-65f0-417b-8b76-b0a2ea079e3b)

**Comment by david-hummingbot** [comment]
Did you try running the docker instance from a different directory? For example in your terminal create a new folder named "hummingbot-test" and then launch the instance from there. There could be some leftover configs from a previous instance causing the issue. 

```
mkdir hummingbot-test
cd hummingbot-test
docker run -it --name hummingbot-instance -v $(pwd)/hummingbot_files:/conf hummingbot/hummingbot:latest
```

**Comment by Omni-NexusAI** [comment]
Both of the instances should each be in their own directories, and the hummingbot dashboard is installed in root directory I believe. I did try running a curl command to see if I get back a valid request, and it also came back invalid. So, I started thinking that it's something with my account, and so I looked through some more of their FAQ on their API keys and I realized that I didn't think that I hadn't funded the account yet as I was simply just setting up for tests. This appears to be the issue the whole time due to my account being new, and because of that my API 's weren't activating yet. After adding funds into the account I managed to connect Kraken now.
